config/deploy/dev.rb

```diff
@@ -0,0 +1 @@
+server 'upaya-dev.18f.gov', roles: %w(web app db)
```

Comment:
Nope, I added that to the [global config](https://github.com/18F/identity-idp/pull/30/files#diff-6dc906be0643182e5bae6aa1506221e6R21) rather than repeating it for each env.